cms/envs/devstack.py

```diff
@@ -162,6 +162,9 @@ def should_show_debug_toolbar(request):  # lint-amnesty, pylint: disable=missing
 ################### FRONTEND APPLICATION COURSE AUTHORING ###################
 COURSE_AUTHORING_MICROFRONTEND_URL = 'http://localhost:2001'
 
+################### FRONTEND APPLICATION DISCUSSIONS ###################
+DISCUSSION_MICROFRONTEND_URL = 'http://localhost:2002'
```

Comment:
What will be the prod url? We'd need to add that as well in the internal settings.

Reply:
The prod URL for MFEs isn't really fixed, it defaults to None so needs to be set in the config file, so it's available in production.py. 

If you mean what would be the production value for the edX instance, I'm not sure. I don't know how/where the discussion MFE is deployed currently. For the sandbox it would be: "https://discuss.discussions.sandbox.opencraft.hosting"

Reply:
it is not deployed on prod currently so I was wondering if there is any name decided for that or not. e.g. `course-authoring.edx.org` maybe its `discussions.edx.org` or something similar. If that has not discussed or sorted out, no worries :)

Reply:
I initially set it to "forum.site-name.org" however, with the way `edx/configuration` works that ran into issues since that name is also used by the ruby service. I'm not sure what name would be used though in the URL though! I don't think it's come up so far.

Reply:
got it thanks.